packages/cli/src/zosjobs/download/download-output/Output.handler.ts

```diff
@@ -35,6 +35,8 @@ export default class OutputHandler extends ZosmfBaseHandler {
         const binary: boolean = this.mArguments.binary;
         const record: boolean = this.mArguments.record;
         const encoding: string = this.mArguments.encoding;
+        const waitForActive: boolean = this.mArguments.wfa;
+        const waitForOutput: boolean = this.mArguments.wfo;
```

Comment:
Please use full names of arguments here. Functionally it is equivalent to using aliases, but makes the code easier to read and maintain in case aliases change in the future.
```suggestion
        const waitForActive: boolean = this.mArguments.waitForActive;
        const waitForOutput: boolean = this.mArguments.waitForOutput;
```